man/os-release.xml

```diff
@@ -411,6 +411,15 @@
           determines the fallback hostname.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ARCHITECTURE=</varname></term>
+          <listitem><para>A string that specifies which CPU architecture the binaries require. The architecture
+          is not exactly in the format reported by <citerefentry><refentrytitle>uname</refentrytitle><manvolnum>2</manvolnum></citerefentry>
```

Comment:
"not exactly"? actually, they are quite different.

Please drop the references to hostname/uname. Just point the user to ConditionArchitecture= in systemd.unit(5) man page

Reply:
Thanks, that's the perfect reference I wanted to have there but failed to come up with ;) Will do.